### PR TITLE
DOCS-1928: Update get started page

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Review by JessamyT — APPROVED** [review]
LGTM % two small comments

File: docs/get-started/_index.md [hunk 1]
```diff
@@ -3,61 +3,29 @@ title: "Get Started"
...
+{{% card link="/get-started/try-viam/" %}}
+{{< /cards >}}
+
 <br>
+If you're ready to try it on your own machine, start by installing `viam-server`.
+If you need any help along the way, join our community:
+
+{{< cards >}}
+{{% card link="/get-started/installation/" %}}
+{{% manualcard link="https://discord.gg/viam" %}}
```
Comment (npentrel) [inline docs/get-started/_index.md:22]:
```suggestion
<!-- markdownlint-disable-next-line MD034 -->
{{% manualcard link="https://discord.gg/viam" %}}
```

File: docs/get-started/_index.md [hunk 2]
```diff
@@ -3,61 +3,30 @@ title: "Get Started"
...
+{{% card link="/get-started/try-viam/" %}}
+{{< /cards >}}
+
 <br>
+If you're ready to try it on your own machine, start by installing `viam-server`.
+If you need any help along the way, join our community:
+
+{{< cards >}}
+{{% card link="/get-started/installation/" %}}
+<!-- markdownlint-disable-next-line MD034 -->
```
Comment (npentrel) [inline docs/get-started/_index.md:22]:
```suggestion
```

File: docs/get-started/_index.md [hunk 3]
```diff
@@ -3,61 +3,30 @@ title: "Get Started"
...
+
+{{< cards >}}
+{{% card link="/get-started/viam/" %}}
+{{% card link="/get-started/try-viam/" %}}
+{{< /cards >}}
+
 <br>
+If you're ready to try it on your own machine, start by installing `viam-server`.
+If you need any help along the way, join our community:
+
```
Comment (npentrel) [inline docs/get-started/_index.md:19]:
```suggestion

<!-- markdownlint-disable-file MD034 -->

```

**Pull request merged**